extension/src/testConfigManager.ts

```diff
@@ -11,46 +12,45 @@ import { TestConfig } from './Models/testConfig';
 import * as Logger from './Utils/Logger/logger';
 
 export class TestConfigManager {
-    constructor(private readonly _configPath: string, private readonly _projectManager: ProjectManager) {
+    private readonly _configPath: string;
+    constructor(private readonly _projectManager: ProjectManager) {
+        const workspaceFolders = workspace.workspaceFolders;
+        if (!workspaceFolders) {
+            throw new Error('Not supported without a folder!');
+        }
+        this._configPath = path.join(workspaceFolders[0].uri.fsPath, '.vscode', Configs.TEST_LAUNCH_CONFIG_NAME);

```

Comment:
How about multiple root scenario?  Always pick up the first one?

Reply:
yes , I would always store it in the first one considering the config for different projects are merged in one config